src/main/scala/com/raphtory/core/deploy/RaphtoryService.scala

```diff
@@ -9,6 +9,46 @@ import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe.TypeTag
 
+/**
+  * {s}`RaphtoryService[T]`
+  *
+  *  : `RaphtoryService` for initialising spout, graphbuilder, deploying spout, partition and query manager
+  *
+  * ## Methods
+  *
+  *    {s}`defineSpout(): Spout[T]`
+  *      : Initialise Spout
+  *
+  *    {s}`defineBuilder(): GraphBuilder[T]`
+  *      : Initialise GraphBuilder
+  *
+  *    {s}`spoutDeploy(): Unit`
+  *      : Deploy spout
```

Comment:
yes this deploys the spout, but how? what function does it call? where does it get the spout from?